tests/test_tools/app.config.test

```diff
@@ -0,0 +1,5 @@
+CONFIG_MODULE_DUMMY_THREAD=y
+CONFIG_MODULE_SHELL=y
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=n
+CONFIG_SHELL_NO_ECHO=y
+CONFIG_SHELL_NO_PROMPT=y
```

Comment:
Now that this application has a configuration file, it can be added to the murdock list https://github.com/RIOT-OS/RIOT/blob/fbe6748c57a72c04c081da0d834bd44a0e1eeafb/.murdock#L27-L29